lendsmartpub/manager
# Added new Tab and Client Id in env file

## Description 📝

**What does this PR do?**

This PR adds .env file with client ID and have added a tab called Yes Done in create PAge

## Preview 📷

**Remove this section or include a screenshot or screen recording of the change**

## How to test 🧪

clone locally and 

yarn install and yarn up

use login details and go to linodes page click create to see the new tab

**What are the steps to reproduce the issue or verify the changes?**

**How do I run relevant unit or e2e tests?**
